Alisha/data/lab2.ipynb

```diff
@@ -0,0 +1,1050 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.    sns.barplot(x=top_countries.values, y=top_countries.index, palette="viridis")
</pre><p>Use matplotlib wherever possible</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/suparthghimire/data-analysis-and-visualization-bca-2025/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>